### PR TITLE
Allow dependencies to be installed in 32 bits bottle

### PR DESCRIPTION
**Pull request by Kinsteen**
# Description
See #2861, couldn't update because I don't have write permissions anymore.

I didn't test, but you can look at the previous PR to know what it does and how to test it.

Also, an update to the .pot files should be required to have the new string properly translated.

### CODE REVIEW

**Comment by koplo199** [comment]
@orowith2os Any chance to get this PR merged? Seems the previous one was approved: https://github.com/bottlesdevs/Bottles/pull/2861#issuecomment-1685344289

**Comment by orowith2os** [comment]
@koplo199 needs testing, otherwise I can look at merging it.

File: bottles/frontend/views/bottle_dependencies.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def __init__(self, details, config: BottleConfig, **kwargs):
         self.btn_report.connect("clicked", open_doc_url, "contribute/missing-dependencies")
         self.btn_help.connect("clicked", open_doc_url, "bottles/dependencies")
 
-        if self.manager.utils_conn.status == False:
-            self.stack.set_visible_child_name("page_offline")
+        if not self.manager.utils_conn.status:
+            self.stack.set_visible_child_name("page_offline")
```
Comment (koplo199) [inline bottles/frontend/views/bottle_dependencies.py:68]:
```suggestion
            self.stack.set_visible_child_name("page_offline")
```
Remove space at end of line
```suggestion
            self.stack.set_visible_child_name("page_offline")
```
```suggestion
            self.stack.set_visible_child_name("page_offline")
```

File: bottles/frontend/views/bottle_dependencies.py [hunk 2]
```diff
@@ -63,8 +63,8 @@ def __init__(self, details, config: BottleConfig, **kwargs):
...
```
Comment (koplo199) [inline bottles/frontend/views/bottle_dependencies.py:67]:
```suggestion
            self.stack.set_visible_child_name("page_offline")
```
Remove space at end of line
```suggestion
            self.stack.set_visible_child_name("page_offline") 
```

**Comment by koplo199** [comment]
Seems the part handling the `for` verb from the previous [PR](https://github.com/bottlesdevs/Bottles/pull/2861/files#diff-99e86322ca2c3d04a795835ce5bb4e39ab03f1000e2959858c98c33a32b2d56d) is missing:  
https://github.com/bottlesdevs/Bottles/blob/208b8b77266f1f3d630b0ba3f9fae87953af7ff9/bottles/backend/managers/dependency.py#L130-L133
(Apparently GitHub does not allow suggesting change to unmodified files, so I cannot leave an suggestion there)

**Comment by koplo199** [comment]
It also seems the `remove_dependency` did not benefit from the `widget` to `_widget` renaming:
https://github.com/bottlesdevs/Bottles/blob/6133d894cb6fec1ff47d6de1046e9cc355acc2de/bottles/frontend/widgets/dependency.py#L152-L157
I also can't leave a GitHub suggestion there, for the same reason.

**Comment by koplo199** [comment]
Fixes https://github.com/bottlesdevs/Bottles/issues/2918
Fixes https://github.com/bottlesdevs/Bottles/issues/2166

**Comment by koplo199** [comment]
@Kinsteen Sorry to ping you directly, I assume you've been busy in life lately.

Hope you don't mind if I implement the requested changes above so we could merge soon. I'll do so by forking your branch [32bits-dependencies](https://github.com/Kinsteen/Bottles/tree/32bits-dependencies), meaning you'll of course retain full authorship of the commits present in there.

**Comment by Kinsteen** [comment]
@koplo199 I have dropped working on Bottles, I would gladly appreciate if you could do the modifications :)

**Comment by orowith2os** [comment]
Closing in favor of https://github.com/bottlesdevs/Bottles/pull/3101

**Pull request closed without merging**